docs/source/api_reference/pipeline_schema_constants/testing.rst

```diff
@@ -0,0 +1 @@
+.. automodule:: easylink.pipeline_schema_constants.testing
```

Comment:
I just didn't like the name "tests" - it kept tricking me into thinking that somehow there were pytests defined there or something